FEATURE_FLAGS.md

```diff
@@ -0,0 +1,64 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+# Superset Feature Flags
+*This is a list of the current Superset optional features. See config.py for default values. These features can be turned on/off by setting your preferred values in superset_config.py to True/False respectively*
+
+## In Development
+*These features are considered **unfinished** and should only be used on development environments.*
+
+- **CLIENT_CACHE** *Introduces a client (browser) cache*
+- **DISABLE_DATASET_SOURCE_EDIT**
+- **DYNAMIC_PLUGINS**
+- **ENABLE_EXPLORE_JSON_CSRF_PROTECTION**
+- **ENABLE_TEMPLATE_PROCESSING**
+- **KV_STORE**
+- **PRESTO_EXPAND_DATA** *Exposes API endpoint to compute thumbnails*
```

Comment:
This actually expands complex types (arrays, objects) in SQL Lab, exploding them into extra columns and rows.

Reply:
thanks for catching! Copy pasta.. I moved it down to Thumbnails.